Gemfile

```diff
@@ -53,7 +53,7 @@ gem 'rack-timeout', require: false
 gem 'redacted_struct'
 gem 'redis', '>= 3.2.0'
 gem 'redis-namespace'
-gem 'redis-session-store', '>= 0.11.3'
+gem 'redis-session-store', github: '18f/redis-session-store', tag: 'v0.11.4-18f'
```

Comment:
I know we have the pattern up above with `@hostdata_gem ||=` above to allow for local development, but we really don't iterate with this gem often so I figured it was easier to use the simpler form like this